cobra/flux_analysis/sampling.py

```diff
@@ -637,14 +681,17 @@ def sample(self, n, fluxes=True):
             n = n_process * self.np
             # No with statement or starmap here since Python 2.x
             # does not support it :(
-            mp = Pool(self.np)
+            mp = Pool(self.np, initializer=mp_init, initargs=(self,))
             chains = mp.map(
                 _sample_chain,
-                zip([self] * self.np, [n_process] * self.np, range(self.np)),
+                zip([n_process] * self.np, range(self.np)),
                 chunksize=1)
```

Comment:
Have you played with `imap_unordered` and a larger chunksize at some point? Or is the communication cost negligible compared to in-process computation time?